lib/webpacker/railtie.rb

```diff
@@ -29,7 +29,7 @@ class Webpacker::Engine < ::Rails::Engine
   end
 
   initializer "webpacker.bootstrap" do
-    if defined?(Rails::Server) || defined?(Rails::Console)
+    if (defined?(Rails::Server) || defined?(Rails::Console)) && Webpacker.config.config_path.exist?
```

Comment:
Why we need to check config file? The idea is to fail when running server or console if none exist. For other things like running react on rails installer this block won't run.